src/gui/AutomationPatternView.cpp

```diff
@@ -375,19 +374,21 @@ void AutomationPatternView::paintEvent( QPaintEvent * )
 	const int lineSize = 3;
 	p.setPen( c.darker( 300 ) );
 
-	for( bar_t t = 1; t < width() - TCO_BORDER_WIDTH; ++t )
```

Comment:
The variable names "t" and "tx" are leftovers from when "tact" was used instead of "bar", which was fixed in #4878.